natlas-server/app/scope/ipscanmanager.py

```diff
@@ -1,50 +1,84 @@
 from netaddr import IPSet, IPNetwork
-from app.cyclicprng import CyclicPRNG
+from app.scope.cyclicprng import CyclicPRNG
 
 
 class IPScanManager:
 	networks = []
 	total = 0
 	rng = None
 
+	def __init__(self):
+		self.networks = []
+		self.total = 0
+		self.rng = None
+
 	def __init__(self, whitelist, blacklist):
-		ipset = IPSet([])
 		self.networks = []
 		self.total = 0
 		self.rng = None
 
+		self.setWhitelist(whitelist)
+		self.setBlacklist(blacklist)
+		self.initializeManager()
+
+	def setWhitelist(self, whitelist):
+		self.white = IPSet([])
```

Comment:
Can you update these to self.whitelist and self.blacklist? They are called `setWhitelist` and `setBlacklist` so it's strange that the variables they are modifying are not `whitelist` and `blacklist`. (It also just occurred to me that we might want to change our terminology at the project level around these concepts, as I know that `whitelist` and `blacklist` are not the best terms to use really.)